specification/cognitiveservices/data-plane/TranslatorText/preview/v1.0-preview.1/TranslatorBatch.json

```diff
@@ -1000,6 +1066,11 @@
           "description": "Number of cancelled",
           "type": "integer",
           "example": 0
+        },
+        "totalCharacterCharged": {
```

Comment:
what encoding is used to calculate the character count?

Reply:
replied at the meeting.  Krishna will clarify